Wiki/Software/Beginner-Windows.md

```diff
@@ -0,0 +1,215 @@
+3. You have computer running 64-bit Windows.
+      * As of this writing, no platforms other than 64-bit Windows are supported.
+4. ***Windows 11 has problems that are out of our control. Do not use Windows 11. Downgrade to Windows 10 if possible.***
+5. Your computer must be sufficiently powerful:
+      - If you intend to control **1** Switch: A dual-core processor @ 3 GHz no older than 2015 should be sufficient.
+      - If you intend to control **2** Switches: A quad-core processor @ 3 GHz no older than 2015 should be sufficient.
+      - If you intend to control **4** Switches: 6-8 cores minimum.
+
+
+## Step 1: HARDWARE ASSEMBLY
+
+Once you have your hardware, you need to make some connections.
+
+Make the following connections:
+- UART cable green (TX) to Arduino pin1 (TX -> 1)
+- UART cable white (RX) to Arduino pin0 (RX <- 0)
+- UART cable black (GND) to Arduino GND (any one is fine)
+- UART cable red (VCC) – leave unconnected
+
+<img src="/Wiki/Hardware/images/leonardo-0.jpg" height="600">
+
+
+## Step 2: Download the SerialPrograms package.
+
+1. Download the latest version of our programs from [here](https://github.com/PokemonAutomation/ComputerControl/releases).
+
+> (The link should look like something like `PA-SerialPrograms-0.x.x-xxxxxxxx.zip`)
+
+   * If you get a virus or malware warning, ignore it. These are known false positives. If you don't trust us, the [source code is here](https://github.com/PokemonAutomation/Arduino-Source/tree/main/SerialPrograms).
+
+2. Once you have downloaded the package, unzip to somewhere you can access later. **Do not put it on Microsoft OneDrive.**
+
+## Step 3: Flash PABotBase into your Device.
+
+The root folder of the SerialPrograms package should have a set of .hex files for each of the different devices.
+
+<img src="/Wiki/Software/images/serial-programs-root.png" height="300">
+
+1. Run the QMK Toolbox program that you downloaded in PART 3.
+2. Open the .hex named `PABotBase-Switch-20211025-ArduinoLeonardo.hex` located in the roof of the folder you downloaded and unzipped.
+3. Change the MCU to `atmega32u4`.
+4. Check the "Auto-Flash" box.
+
+<img src="https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Software/images/tutorial-windows-leonardo-2.png" height="600">
+
+5. Plug the Leonardo into your computer.
+6. Press the reset button.
+
+<img src="https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Software/images/tutorial-windows-leonardo-3.jpg" height="400">
+
+The QMK program will now flash the program to the Leonardo and show a bunch of logging.
+Afterwards, the 3 LEDs on the Leonardo should flash in unison for 5 seconds before turning off.
+
+<img src="https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Software/images/tutorial-windows-leonardo-4.png" height="600">
```

Comment:
This image will need to be updated since the file name is for TurboA instead of PABotBase.

Reply:
Do we have a file for this? I'm not seeing it in the image folder for hardware or software.

Reply:
We don't. lol